README.md

```diff
@@ -98,6 +98,45 @@ umm_var = get_all_umm_var(var_info)
 export_all_umm_var_to_json(list(umm_var.values()), output_dir='local_dir')
 ```
 
+### End-to-end UMM-Var generation and publication:
+
+```
+from cmr import CMR_OPS
+from varinfo import generate_collection_umm_var
```

Comment:
Is this actually importable from the package.  I didn't see it in __init__.py and I had to import it from varinfo.generate_umm_var

When I built the package and installed with `pip install -e .`. I got this error

```
>>> from varinfo import generate_collection_umm_var
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
ImportError: cannot import name 'generate_collection_umm_var' from 'varinfo' (/Users/savoie/projects/data-services/earthdata-varinfo/varinfo/__init__.py)
```

Reply:
Yeah - good catch. I had added it temporarily to `__init__.py` so this would work. But then it turns out that caused a circular import issue and I reverted it (but not the documentation).

- [ ] Fix README